rollup/internal/config/l2.go

```diff
@@ -29,15 +29,15 @@ type ChunkProposerConfig struct {
 	MaxBlockNumPerChunk             uint64  `json:"max_block_num_per_chunk"`
 	MaxTxNumPerChunk                uint64  `json:"max_tx_num_per_chunk"`
 	MaxL1CommitGasPerChunk          uint64  `json:"max_l1_commit_gas_per_chunk"`
-	MaxL1CommitCalldataSizePerChunk uint64  `json:"max_l1_commit_calldata_size_per_chunk"`
+	MaxL1CommitCalldataSizePerChunk uint32  `json:"max_l1_commit_calldata_size_per_chunk"`
```

Comment:
Any reason to change this?

Reply:
because db type is `uint32`, just for consistency.

Reply:
will change all the types to uint64, while keeping this type in db schema as an integer, in normal cases, this will not cause any problem.